Samples/Word-Get-Started-With-Dev-Kit/src/taskpane/taskpane.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+/* global document, Office, Word */
+
+Office.onReady((info) => {
+  if (info.host === Office.HostType.Word) {
+    document.getElementById("sideload-msg").style.display = "none";
+    document.getElementById("app-body").style.display = "flex";
+    document.getElementById("run").onclick = run;
+  }
+});
+
+export async function run() {
+  return Word.run(async (context) => {
+    /**
+     * Insert your Word code here
```

Comment:
I'm not sure of the value of this comment. Technically, you'd most likely want to make a new function and connect that to a new HTML control.
```suggestion
     * Insert your Word code here.
```